pocketlint/__init__.py

```diff
@@ -37,11 +37,17 @@ class PocketLintConfig(object):
     """Configuration object that a project should use to tell pylint how
        to operate.  Instance attributes:
 
-       falsePositives -- A list of FalsePositive objects for filtering
-                         incorrect pylint error messages.
+       falsePositives      -- A list of FalsePositive objects for filtering
+                              incorrect pylint error messages.
+                              Default: []
+       load_all_extensions -- Load all the extensions to the pylint. This will resolve
+                              I1101(c-extension-no-member) messages but it could be
+                              potentially dangerous. See man pylint --unsafe-load-any-extension
+                              Default: True
     """
     def __init__(self):
         self.falsePositives = []
+        self.load_all_extensions = True
```

Comment:
It might be better to use `loadAllExtensions` here. I know camel case names are not pep8, but we are already using camel case here so it would be better to keep it for new properties.

Reply:
Fixed.